zip-0213.rst

```diff
@@ -0,0 +1,169 @@
+::
+
+  ZIP: 213
+  Title: Shielded Coinbase
+  Owners: Jack Grigg <jack@z.cash>
+  Status: Draft
+  Category: Consensus
+  Created: 2019-03-30
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST" and "MAY" in this document are to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+The term "network upgrade" in this document is to be interpreted as described in ZIP 200
+[#zip-0200]_.
+
+The term "Sapling" in this document is to be interpreted as described in ZIP 205
+[#zip-0205]_.
+
+The terms "Founders' Reward" and "funding stream" in this document are to be interpreted
+as described in ZIP 207 [#zip-0207]_.
```

Comment:
Since ZIP 207 is definitely not included in Heartwood and there will be a new ZIP (or ZIP 1014 will be amended) to include the dev funding in NU4, I think we should just simplify this ZIP by removing references to ZIP 207.

Reply:
I considered that, but it's still relevant to the rationale. If we weren't going to have funding stream outputs ever be shielded, then we wouldn't technically need to require that all outputs be decryptable.